futures-channel/src/mpsc/mod.rs

```diff
@@ -1020,9 +1020,6 @@ impl<T> Receiver<T> {
     /// It is not recommended to call this function from inside of a future,
     /// only when you've otherwise arranged to be notified when the channel is
     /// no longer empty.
-    ///
-    /// This function will panic if called after `try_next` or `poll_next` has
-    /// returned `None`.
```

Comment:
Seems this is a doc of Receiver::try_next. Could you also update a doc of UnboundedReceiver::try_next?